### PR TITLE
Xspec models lite

### PR DESCRIPTION
**Pull request by grburgess**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by grburgess** [comment]
The tarball to be downloaded is very large and causing the builds to timeout. Is there anyway around this?

**Comment by grburgess** [comment]
The builds are timing out only on travis due to the long build time. Otherwise, I think we are ok!

**Comment by grburgess** [comment]
@conda-forge/help-c-cpp is it ok that the timeout occurs on travis?

**Comment by chrisburr** [comment]
Yeah that's fine as the feedstock will be built on Azure.

File: recipes/xspec-models-lite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{% set name = "xspec-models-lite" %}
+{% set version = "6.22" %}
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]: From the URL it looks like the version is 6.22.1?
```suggestion
{% set version = "6.22.1" %}
```

File: recipes/xspec-models-lite/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft6.22.1/xspec-modelsonly-v6.22.1.tar.gz
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]: Using jinja2 variables can avoid mistakes when updating:
```suggestion
  url: https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft{{ version }}/xspec-modelsonly-v{{ version }}.tar.gz
```

File: recipes/xspec-models-lite/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  number: 0
+
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cfitsio
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]: I think the build requirements should be:
```suggestion
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('fortran') }}
    - make
```
The others look like `host` and `run` dependencies.

File: recipes/xspec-models-lite/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    - xorg-libx11   # [osx]
+
+  host:
+    - ncurses 5.*
+    - readline
+    - perl
+    - cfitsio
+    - ccfits
+    - wcslib
+    - xorg-libx11  # [osx]
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]:
```suggestion
    - xorg-libx11
```

File: recipes/xspec-models-lite/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+  run:
+    - ncurses
+    - readline
+    - perl
+    - cfitsio
+    - ccfits
+    - wcslib
+    - xorg-libx11  # [osx]
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]:
```suggestion
    - xorg-libx11
```

File: recipes/xspec-models-lite/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+
+test:
+
+  commands:
```
Comment (chrisburr) [inline recipes/xspec-models-lite/meta.yaml]: Can you add some kind of test for macOS? Without one conda will happily build an empty package if it doesn't realise the build failed.

File: recipes/xspec-models-lite/startup.sh [hunk 7]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```
Comment (chrisburr) [inline recipes/xspec-models-lite/startup.sh]: What is this file for?

Comment (grburgess) [reply recipes/xspec-models-lite/startup.sh]: A leftover from building locally. It is removed now.

File: recipes/xspec-models-lite/meta.yaml (hunk 6 above)
Comment (grburgess) [reply recipes/xspec-models-lite/meta.yaml]: Done :)

**Comment by grburgess** [comment]
The libraries are needed during the build phase because Xspec is "fun"

**Comment by grburgess** [comment]
@chrisburr is there an issue with the cfitsio library being included?
